tests/js/spec/components/modals/recoveryOptionsModal.spec.jsx

```diff
@@ -19,8 +18,9 @@ describe('RecoveryOptionsModal', function () {
     });
     wrapper = mountWithTheme(
       <RecoveryOptionsModal
-        Body={Modal.Body}
-        Header={Modal.Header}
+        Body={p => p.children}
+        Header={p => p.children}
+        Footer={p => p.children}
```

Comment:
I'm not familiar with this. In this case, what's `p.children`?

Reply:
This is just a do-nothing component that will pass along whatever the modal declares as it's heading content. In this case we don't really care what the header is.

In the future it might make sense to have a fixture for this, especially for acceptance tests.